source/codegen/metadata/nixnet/functions.py

```diff
@@ -0,0 +1,1539 @@
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbGetPropertySize': {
+        'cname': 'nxdbGetPropertySize',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'dbObjectRef',
+                'type': 'nxDatabaseRef_t'
+            },
+            {
+                'direction': 'in',
+                'enum': 'DBProperties',
+                'name': 'propertyID',
+                'type': 'u32'
+            },
+            {
+                'direction': 'out',
+                'name': 'propertySize',
+                'type': 'u32'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbMerge': {
+        'cname': 'nxdbMerge',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'targetClusterRef',
+                'type': 'nxDatabaseRef_t'
+            },
+            {
+                'direction': 'in',
+                'name': 'sourceObjRef',
+                'type': 'nxDatabaseRef_t'
+            },
+            {
+                'direction': 'in',
+                'enum': 'CopyMode',
+                'name': 'copyMode',
+                'type': 'u32'
+            },
+            {
+                'direction': 'in',
+                'name': 'prefix',
+                'type': 'char[]'
+            },
+            {
+                'direction': 'in',
+                'name': 'waitForComplete',
+                'type': 'u32'
+            },
+            {
+                'direction': 'out',
+                'name': 'percentComplete',
+                'type': 'u32'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbOpenDatabase': {
+        'cname': 'nxdbOpenDatabase',
+        'init_method': True,
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'databaseName',
+                'type': 'const char[]'
+            },
+            {
+                'direction': 'out',
+                'name': 'databaseRef',
+                'type': 'nxDatabaseRef_t'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbRemoveAlias': {
+        'cname': 'nxdbRemoveAlias',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'databaseAlias',
+                'type': 'const char[]'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbSaveDatabase': {
+        'cname': 'nxdbSaveDatabase',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'databaseRef',
+                'type': 'nxDatabaseRef_t'
+            },
+            {
+                'direction': 'in',
+                'name': 'dbFilepath',
+                'type': 'const char[]'
+            }
+        ],
+        'returns': 'nxStatus_t'
+    },
+    'DbSetProperty': {
+        'cname': 'nxdbSetProperty',
+        'codegen_method': 'no',
```

Comment:
How are you tracking the `codegen_method: no` stuff? Maybe a work item and a comment in scrapigen? (you might already have this, I can't remember).

Reply:
I have created tasks on AzDO for tracking remaining API based on different categories (new size mechanism, void return types etc.). We were also planning to update scrapigen to automatically tag these APIs as codegen-no through a hardcoded list of disabled APIs and work our way through till that list is empty.